master/docs/manual/cfg-schedulers.rst

```diff
@@ -811,22 +811,27 @@ What you need in your config file is something like::
                  name="force",
                  builderNames=["my-builder"],
 
-                 # will generate a combo box
-                 branch=util.ChoiceStringParameter(name="branch",
-                                                   choices=["main","devel"],
-                                                   default="main"),
+                 codebases = [ util.CodebaseParameter(
+                     "",
+                     name = "Main repository",
+                     # will generate a combo box
+                     branch=util.ChoiceStringParameter(
+                         name="branch",
+                         choices=["master", "hest", primaryBranch],
+                         default="master"),
+
+                         # will generate nothing in the form, but revision, repository,
+                         # and project are needed by buildbot selfcheduling system so we
```

Comment:
I think comment is garbled here, take a look at previous version if this comment.